lib/asset/serve-snap/src/Obelisk/Asset/Serve/Snap.hs

```diff
@@ -94,8 +95,15 @@ serveAsset' doRedirect base fallback p = do
           let finalFilename = base </> p </> "encodings" </> T.unpack (decodeUtf8 e)
           stat <- liftIO $ getFileStatus finalFilename
           modifyResponse $ setHeader "Last-Modified" "Thu, 1 Jan 1970 00:00:00 GMT"
-          modifyResponse $ setResponseCode 200 . setContentLength (fromIntegral $ fileSize stat) . setContentType (fileType defaultMimeTypes p)
-          sendFile finalFilename
+          modifyResponse $ setHeader "Accept-Ranges" "bytes"
+          modifyResponse $ setContentType (fileType defaultMimeTypes p)
```

Comment:
Out of curiosity, why aren't we composing all these `modifyResponse` into one ?

Reply:
No reason. Will refactor this.